prefetch.bs

```diff
@@ -863,6 +863,16 @@ The \`<code><dfn http-header>Sec-Speculation-Tags</dfn></code>\` HTTP request he
 
 The header field is an [[RFC9651]] Structured Header whose value must be a [=structured header/List=]. The list can contain either [=structured header/Token=] or [=structured header/String=] values. String values represent developer-provided tags, whereas token values represent predefined tags. As of now, the only predefined tag is `null`, which indicates a speculative navigation request with no developer-defined tag.
 
+<h2 id="interaction-with-other-specs">Interaction with other specifications and concepts</h2>
```

Comment:
Let's make this section "Clear Site Data patches" and put it below "Navigation Timing patches".